client/my-sites/plugins/index.js

```diff
@@ -70,7 +70,7 @@ module.exports = function() {
 			)
 		) );
 
-		page( '/plugins/:plugin/:business_plugin?/:site_id?',
```

Comment:
left lingering after 4631897912a8d8876bf4a08d3fa905902f00f134?

cc: @Tug you think it's safe to remove this?